pkg/controller/pxc/tls.go

```diff
@@ -135,6 +136,7 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
 				Kind:  issuerKind,
 				Group: issuerGroup,
 			},
+			Duration: &metav1.Duration{Duration: time.Hour * 24 * 365},
```

Comment:
why do we need this line ? as I remember cert manager issues certs for 3 months and after that they will be recreated.